simplq/src/components/common/QR_Scanner/QR_Scanner.js

```diff
@@ -0,0 +1,76 @@
+//Component responsible for the /scanQr route
+
+import React, { Component } from 'react';
+import QrReader from 'react-qr-scanner';
+import styles from "./QR_Scanner.module.scss";
```

Comment:
Unless it's an accepted convention, let's use `QrScanner.module.scss` and `QrScanner.js` for the filenames and `QrScanner` for the component name. 

I am not very familiar with JS/React naming conventions, so please consult them if required.